lib/cocoapods/generator/xcconfig/xcconfig_helper.rb

```diff
@@ -212,7 +212,7 @@ def self.add_library_build_settings(library_path, xcconfig, sandbox_root, includ
           name = File.basename(library_path, extension).sub(/\Alib/, '')
           dirname = '${PODS_ROOT}/' + library_path.dirname.relative_path_from(sandbox_root).to_s
           build_settings = {
-            'LIBRARY_SEARCH_PATHS' => '$(inherited) ' + quote([dirname]),
```

Comment:
This matches the same behavior as `add_framework_build_settings` above which was *not* adding `$(inherited)`